crates/epaint/src/text/fonts.rs

```diff
@@ -254,6 +254,34 @@ pub struct FontDefinitions {
     pub families: BTreeMap<FontFamily, Vec<String>>,
 }
 
+#[derive(Debug, Clone)]
+pub struct FontInsert {
+    /// Font name
+    pub name: String,
+
+    /// A `.ttf` or `.otf` file and a font face index.
+    pub data: FontData,
+
+    /// Sets the font family
+    pub family: FontFamily,
+
+    /// If append is true & there are multiple fonts for that font family the font will be added as fallback.
+    /// This decides if the font is added to the front or the back.
+    /// So the first font is the primary, and then comes a list of fallbacks in order of priority.
+    pub family_append: bool,
```

Comment:
Maybe we can make this an enum instead, for better clarify:

```
pub enum FontPriority {
    /// Prefer this font before all existing ones.
    ///
    /// If a desired glyph exists in this font, it will be used.
    Highest,

    /// Use this font as a fallback, after all existing ones.
    ///
    /// This font will only be used if the glyph is not found in any of the previously installed fonts.
    Lowest,
}
```